src/main/resources/carpet.mixins.json

```diff
@@ -208,10 +208,10 @@
     "CaveScalerMixin_scarpetMixin",
 
     "BlockEntity_movableBEMixin",
-    "PistonBlock_movableTEMixin",
-    "PistonBlockEntity_movableTEMixin",
-    "World_movableTEMixin",
```

Comment:
yeah, please revert rename of these. These could be renamed in the consequent PR using proper class names from mojmaps as well.